tests/check_sssd_with_tlog.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Check with-tlog feature exists
+  command: authselect list-features sssd
+  register: __tlog_authselect_features
+  changed_when: false
+
+- name: Check if with-tlog authselect feature is currently enabled
+  command: authselect current
+  register: __tlog_authselect_current
+  changed_when: false
+  failed_when: __tlog_authselect_current.rc not in [0, 2]
+
+- name: Read nsswitch.conf
+  slurp:
+    src: /etc/nsswitch.conf
+  register: __nsswitch_slurp
+
+- name: Decode nsswitch content
```

Comment:
since you are using `vars` below, you can get rid of this task

Reply:
Fixed.